en/components/icon-service.md

```diff
@@ -154,19 +157,16 @@ Starting with version 18.1.x of Ignite UI for Angular, we added a new `setFamily
 | default     | arrow_downward | material      | arrow_downward       | Grid, Excel Style Filtering                             | All                       |
 | default     | today          | material      | today                | Date Picker                                             | All                       |
 | default     | date_range     | material      | date_range           | Date Range Picker                                       | All                       |
-| default     | expand         | material      | arrow_drop_down      | Grid                                                    | All                       |
+| default     | expand         | material      | arrow_drop_down      | Grid, TreeNode                                          | All                       |
```

Comment:
Um there seem to be two default expand entries, this one with `arrow_drop_down` is the one that doesn't exist in code from what I can see, so the Grid and Tree usage are with `expand_more`.